tests/unit/user_registry/test_registry.py

```diff
@@ -386,27 +396,27 @@ async def test_get_selected_ivas_with_user():
     jane = john.model_copy(update={"id": ID_OF_JANE})
     registry.dummy_users.append(jane)
     add_iva = registry.add_dummy_iva
-    add_iva(value="123", user_id=john.id)
-    add_iva(value="456", user_id=jane.id)
-    add_iva(value="789", user_id=john.id, state=IvaState.VERIFIED)
+    add_iva(value=PHONE_OF_JOHN, user_id=john.id)
+    add_iva(value=PHONE_OF_JANE, user_id=jane.id)
+    add_iva(value=PHONE_OF_JAMES, user_id=john.id, state=IvaState.VERIFIED)
     get_ivas = registry.get_ivas_with_users
     ivas = await get_ivas(user_id=jane.id)
     assert len(ivas) == 1
-    assert ivas[0].value == "456"
+    assert ivas[0].value == PHONE_OF_JANE
     ivas = await get_ivas(state=IvaState.VERIFIED)
     assert len(ivas) == 1
-    assert ivas[0].value == "789"
+    assert ivas[0].value == PHONE_OF_JAMES
     ivas = await get_ivas(state=IvaState.UNVERIFIED, user_id=john.id)
     assert len(ivas) == 1
-    assert ivas[0].value == "123"
+    assert ivas[0].value == PHONE_OF_JOHN
     assert not registry.published_events
 
 
 async def test_delete_existing_iva():
     """Test deleting an existing IVA."""
     registry = MockUserRegistry()
-    registry.add_dummy_iva(value="123")
-    registry.add_dummy_iva(value="456", user_id=ID_OF_JANE)
+    registry.add_dummy_iva(value=PHONE_OF_JOHN)
```

Comment:
```suggestion
    registry.add_dummy_iva()
```
`PHONE_OF_JOHN` is already the default value